parlai/mturk/webapp/server.py

```diff
@@ -304,16 +319,158 @@ def get(self, worker_target):
+            task_instructions = t.task_config['task_description']
+        except ImportError:
+            try:
+                # Try to find the task in local tasks
+                t = importlib.import_module(find_location_internal)
+                task_instructions = t.task_config['task_description']
+            except ImportError:
+                task_instructions = None
+
+        data = {
+            'assignment_details': assignment,
+            'assignment_content': assignment_content,
+            'assignment_instructions': task_instructions,
+        }
+
+        self.write(json.dumps(data))
+
+
+class ApprovalHandler(BaseHandler):
+    def initialize(self, app):
+        self.mturk_manager = app.mturk_manager
+
+    def post(self, assignment_target):
+        self.mturk_manager.approve_work(assignment_target)
+        data = {
+            'status': True,
+        }
+        self.write(json.dumps(data))
+
+
+class ReverseHandler(BaseHandler):
+    def initialize(self, app):
+        self.mturk_manager = app.mturk_manager
+
+    def post(self, assignment_target):
+        self.mturk_manager.approve_work(assignment_target, True)
+        print('Assignment {} had rejection reversed'.format(assignment_target))
+        data = {
+            'status': True,
+        }
+        self.write(json.dumps(data))
+
+
+class RejectionHandler(BaseHandler):
+    def initialize(self, app):
+        self.mturk_manager = app.mturk_manager
+
+    def post(self, assignment_target):
+        data = tornado.escape.json_decode(self.request.body)
+        reason = data['reason']
+        self.mturk_manager.reject_work(assignment_target, reason)
+        print('Rejected {} for reason {}'.format(assignment_target, reason))
+        data = {
+            'status': True,
+        }
+        self.write(json.dumps(data))
+
+
+class BlockHandler(BaseHandler):
+    def initialize(self, app):
+        self.mturk_manager = app.mturk_manager
+
+    def post(self, worker_target):
+        data = tornado.escape.json_decode(self.request.body)
+        reason = data['reason']
+        self.mturk_manager.block_worker(worker_target, reason)
+        print('Blocked {} for reason {}'.format(worker_target, reason))
+        data = {
+            'status': True,
+        }
+        self.write(json.dumps(data))
+
+
+class BonusHandler(BaseHandler):
+    def initialize(self, app):
+        self.mturk_manager = app.mturk_manager
+
+    def post(self, worker_target):
+        """Requests to /bonus/{worker_id} will give a bonus to that worker.
+        Requires a reason, assignment_id, a unique token (for idempotence),
+        and the bonus amount IN CENTS
+        """
+        data = tornado.escape.json_decode(self.request.body)
+        reason = data['reason']
+        assignment_id = data['assignment_id']
+        bonus_cents = data['bonus_cents']
+        token = data['bonus_token']
+
+        amount = bonus_cents / 100.0
```

Comment:
same here I think even just  `# cents => dollars` or change `amount` var to `dollars` or `dollar_amount`